test/services/submission-request.test.js

```diff
@@ -1,217 +0,0 @@
-const { Application } = require('../../services/application');
```

Comment:
Please, dont delete this unit test.

Reply:
I see. Thanks. But I dont see the unit test 'approveApplication successfully approves an application' in application.test.js

Reply:
I see. This is the new change recently. I need to check once merged. Thanks.